CMakeLists.txt

```diff
@@ -19,6 +19,14 @@
 
 cmake_minimum_required (VERSION 3.13)
 
+project(oneMKL VERSION 0.1.0 LANGUAGES CXX)
+
```

Comment:
For windows project should be defined after default parameters are set (it happens on line 41). We don't need it here

Reply:
Removed